### ISSUE TITLE
tslint not using local tslint.json file

### ISSUE THREAD

**Issue by Anupheaus** [palantir/tslint#3981]
### Bug Report

- __TSLint version__: 5.10.0
- __TypeScript version__: 2.9.1
- __Running TSLint via__: VSCode & CLI

#### TypeScript code being linted

```ts
console.log("hey");
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "no-console": true
    }
}
```

#### Actual behavior
No errors, no warnings, just nothing at all actually.

#### Expected behavior
ERROR: packages/ui/src/dataSource/items.tests.ts[1, 1]: Calls to 'console.log' are not allowed.

I get the error if I explicitly set the config:
`tslint -c ./tslint.json ./packages/ui/src/dataSource/items.tests.ts`
but if I just do this:
`tslint ./packages/ui/src/dataSource/items.tests.ts`
I get nothing returned at all.

**Comment by Anupheaus** [comment]
Sorry, my bad, I didn't realise I had got a `tslint.json` file within a directory above the file I was checking.
